src/element/login-signup.tsx

```diff
@@ -147,7 +149,9 @@ export function LoginSignup({ close }: { close: () => void }) {
     case Stage.Login: {
       return (
         <>
-          <img src={LoginHeader} className="header-image" />
+          <ProgressiveImage src={LoginHeader as string} placeholder={LoginHeaderSmall}>
```

Comment:
I dont think you need this lib, you can use srcset directly in the browser https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/srcset

Reply:
I don't think srcset does progressive image loading, but I can update this PR just to use 1x and 2x images. It should already be a lot better with the decreased sizing.

I think best of all would be progressive image loading with a small, blurred base64 inline image as the preview one. But it might not be necessary to go that far :-)

I will ping you when I have updated the PR

Reply:
Sorry you're right, but progressive image is to do with image encoding, not multiple seperate images: https://stackoverflow.com/questions/40127266/change-jpeg-into-progressive-jpeg-image

I was thinking since a lot of users are on mobile the DPI is usually > 1 so it would make sense to have at least a 1x and 2x image

Reply:
Thanks, I learned something new today :-)